apps/condo/domains/acquiring/schema/RecurrentPaymentContext.js

```diff
@@ -113,12 +113,14 @@ const RecurrentPaymentContext = new GQLListSchema('RecurrentPaymentContext', {
                 const recurrentPaymentContextId = get(existingItem, 'id', null)
                 const sender = get(originalInput, 'sender', null)
                 const dv = get(originalInput, 'dv', null)
+                const autoPayReceipts = get(existingItem, 'autoPayReceipts', null)
 
                 // data vars
                 const newServiceConsumerId = get(originalInput, 'serviceConsumer.connect.id', null)
                 const serviceConsumerId = get(existingItem, 'serviceConsumer', null)
                 const newPaymentDay = get(originalInput, 'paymentDay', null)
                 const paymentDay = get(existingItem, 'paymentDay', null)
+                const newAutoPayReceipts = get(originalInput, 'autoPayReceipts', null)
```

Comment:
Is it correct that this value is taken from `originalInput` and not from `const newItem = { ...existingItem, ...resolvedData }`?

and other cases too?